collectors/icdcm/collector.py

```diff
@@ -59,5 +61,5 @@ def collect(conf, conn):
             if not count % 100:
                 logger.info('Collected %s "%s" conditions', count, record.table)
 
-        except Exception as exception:
-            logger.exception(repr(exception), exc_info=True)
+        except:
+            base.config.SENTRY.captureException()
```

Comment:
These parsing exceptions can be very unhelpful on their own. Does `captureException()` include the traceback by default?